packages/main/src/themes/base/sizes-parameters.css

```diff
@@ -194,11 +194,12 @@
 	--_ui5_rb_label_width: calc(100% - 2rem + 1px);
 	--_ui5_rb_rtl_focus_right: 0.375rem;
 
-	--_ui5_wheelslider_item_width: 4rem;
```

Comment:
Here it is a good idea to set the most common value of the variables, so that you only override them when necessary.(And not in al themes) In this case you can set them to 
```css
--_ui5_wheelslider_item_width: 48px;
--_ui5_wheelslider_item_height: 46px;
```

And only override them for sap_belize, sap_belize_hcb & sap_belize_hcw themes